### DISCUSSION TITLE
About Github Branches

### DISCUSSION THREAD

**Discussion by mrakiyaaa** [community/community#87433] [Repositories] [answered]
### Select Topic Area

Question

### Body

How do you create a new branch in a Git repository using the command line?

**Comment by faridnec** [comment, marked as answer]
To create a new branch in a Git repository using the command line, you can use
```bash
git branch branch_name
```
This command creates a new branch named branch_name but doesn't switch to it. If you want to create and switch to the new branch in a single command, you can use

```bash
git checkout -b branch_name
```
Starting from Git version 2.23, you can also use the git switch command:
```bash
git switch -c branch_name
```

**Comment by ghostinhershell** [comment]
Hey there!  👋

Thanks for posting in the GitHub Community, @mrakiyaaa! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

I've gone ahead and moved this to the correct category for you. Good luck!
